src/help/contribution/extension-development/quickstart.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Source development quickstart
+lang: en-US
+---
+
+# Source development quickstart
+
+## Preparing the development environment
+
+1. Create your own fork of one of our repositories, like [Extensions Promises](https://github.com/Paperback-iOS/extensions-promises).
+
+1. Create a new branch before working on your new fork. The method of branching depends on the Git CLI/GUI that you use, reference your software manual, or contact us if you need help with this step.
+
+1. Clone the fork you've just created on your computer.
+
+1. Install dependencies by running
+   ```bash
+   npm install
+   ```
+
+You are ready to start developing a source. When you think the source should be available for users, submit a pull request to one of our repository.
+
+:::: guide Repositories
+Please note each of our repositories has its own specificity.
+
+Check our [repositories list]() to figure where your source should be, all repositories are available on GitHub.
```

Comment:
Missing URL?
```suggestion
Check our [repositories list](/help/guides/adding-repos) to figure where your source should be, all repositories are available on GitHub.
```

Is this correct?

Reply:
Woupsi 😕

Reply:
It is correct, fixed with an anchor in f4e8d19